source/code-snippets/_example-first-class-function.html.erb

```diff
@@ -46,13 +46,13 @@
 
   $fonts: Tahoma, Geneva, "Helvetica Neue", Helvetica, Arial, sans-serif
 
-  content
```

Comment:
This is a valid CSS selector, but it might be clearer to explicitly make it a class selector.

Reply:
It might be a valid CSS selector, but the element would not be valid HTML markup. Anyway, something more familiar like the suggested class selector makes sense to me.